.github/workflows/helm-test.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "8-add-configmap" ]
+  pull_request:
+    branches: [ "8-add-configmap" ]
```

Comment:
```suggestion
  push:
    branches:
      - main
    tags:
      - v*
  pull_request:
```

Reply:
I try to fix pipeli for my branch and after that we can move to main branch or tag

Reply:
Hey @hamedsalim1999,

Any updates here?

Reply:
Hi @mostafa,

I'm trying to complete this task by the end of this week.